waku/waku_test.go

```diff
@@ -278,7 +278,6 @@ func TestSymKeyManagement(t *testing.T) {
 	var err error
 	var k1, k2 []byte
 	w := New(nil, nil)
-	id1 := string("arbitrary-string-1")
```

Comment:
This is causing tests to fail. 
```
[2020-02-20T11:54:35.582Z] # github.com/status-im/status-go/waku [github.com/status-im/status-go/waku.test]
[2020-02-20T11:54:35.582Z] waku/waku_test.go:283:2: undefined: id1
[2020-02-20T11:54:35.582Z] waku/waku_test.go:288:24: undefined: id1
```

They pass on develop for waku. The tests for Whisper fail on develop (confirmed) and I will fix them.